package-lock.json

```diff
@@ -3397,6 +3397,16 @@
       "integrity": "sha512-Phlt0plgpIIBOGTT/ehfFnbNlfsDEiqmzE2KRXoX1bLIlir4X/MR+zSyBEkL05ffWgnRSf/DXv+WrUAVr93/ow==",
       "dev": true
     },
+    "bindings": {
```

Comment:
Why is package.json modified? We really shouldnt be touching unless we need to, cuz it might break stuff